sktree/_forest.py

```diff
@@ -212,3 +213,31 @@ def fit(self, X, y=None, sample_weight=None):
         # Parameters are validated in fit_transform
         self.fit_transform(X, y, sample_weight=sample_weight)
         return self
+
+    def transform(self, X):
+        from joblib import Parallel
+        from sklearn.utils.fixes import delayed
+
+        results = Parallel(
+            n_jobs=self.n_jobs,
+            verbose=self.verbose,
+            prefer="threads",
+        )(delayed(tree.transform)(X) for tree in self.estimators_)
```

Comment:
Note from base.py

        # non-optimized default implementation; override when a better
        # method is possible for a given clustering algorithm

Reply:
What is this?

Reply:
@adam2392 oh I was just jotting down notes just for my own reference to better understand your PR :laughing: I wasn't at all reviewing your PR. With this particular method, I was looking through how it was being used and I found these comments